jekyll/_cci2/server-3-install-prerequisites.adoc

```diff
@@ -45,8 +45,28 @@ Download and install the following software before continuing:
 | KOTS: https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_linux_amd64.tar.gz[Linux]. 
 | {kotsversion}
 | Replicated Kubernetes Application Management
+
+| https://velero.io/docs/v1.6/basic-install/[Velero CLI] 
+| TBC
+| TBC
```

Comment:
The latest version is v1.7.1, which might mean this URL is already outdated.

Reply:
Maybe I could point here instead? https://github.com/vmware-tanzu/velero/releases

Reply:
@ryanwohara ^

Reply:
@rosieyohannan Sounds good to me! :)